src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import BootstrapAlert from 'react-bootstrap/Alert'
+import Button from 'react-bootstrap/Button'
+
+interface Props {
+  /** Defines the color of the alert. Defaults to primary.
+   * @default "primary"
+   **/
+  color?: 'primary' | 'secondary' | 'success' | 'warning' | 'danger' | 'info' | 'light' | 'dark'
+  /** Defines the title of the alert.*/
+  title?: string
+  /** Defines the message of the alert.*/
+  message?: string
```

Comment:
What do you think about making this be able to support all different types of messages (not just a string)

I think we would give it type `React.Node` and then we would be able to do something like this example: https://react-bootstrap.github.io/components/alerts#links.

Reply:
Agree :+1: